server/repository/repository.go

```diff
@@ -158,26 +156,38 @@ func (s *Server) GetAppDetails(ctx context.Context, q *repositorypkg.RepoAppDeta
 		return nil, err
 	}
 	return repoClient.GetAppDetails(ctx, &apiclient.RepoServerAppDetailsQuery{
-		Repo:     repo,
```

Comment:
I've found a few places in the code where you have to pass values around for it to work correctly, and where unit tests won't go red. This could be one of the places.

Reply:
Thanks for pointing to this method. Noticed that we don't set `Plugins` field here and discovered at field is not required at all. Removed it.